README.md

```diff
@@ -36,15 +40,50 @@ In the European Union, this extension's fields correspond to [eForms BG-702 (Sel
 }
 ```
 
+### Lot
+
+A tender with a single lot where the selection criterion only applies for selecting candidates to be invited to the second stage of the procedure.
+
+```json
+{
+  "tender": {
+    "lots": [
+      {
+        "id": "LOT-0001",
+        "selectionCriteria": {
+          "criteria": [
+            {
+              "description": "A description of the criteria",
+              "type": "suitability",
+              "forReduction": true,
+              "numbers": [
+                {
+                  "number": 30,
```

Comment:
Does it make sense to only have a criteria with a 30% weighting? I would've expected the percentages to add up to 100%, like the examples in the award criteria extension.

Reply:
good point, I'll change this to 100% instead of 30%